db/new_seeds/base/add_mentor_scenarios.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+# generate some bits of data we'll need below first, roughly sticking to the
+# structure from the legacy seeds but this time spanning two cohorts (2021,
+# 2022) and three lead provoders
+
+school_cohort_one = FactoryBot.create(:seed_school_cohort, :cip, :valid, :starting_in_2021)
+induction_programme_one = FactoryBot.create(:seed_induction_programme, :cip, school_cohort: school_cohort_one)
+
+school_cohort_two = FactoryBot.create(:seed_school_cohort, :cip, :valid, :starting_in_2021)
+induction_programme_two = FactoryBot.create(:seed_induction_programme, :fip, school_cohort: school_cohort_two)
+
+school_cohort_three = FactoryBot.create(:seed_school_cohort, :fip, :valid, :starting_in_2021)
+induction_programme_three = FactoryBot.create(:seed_induction_programme, :cip, school_cohort: school_cohort_three)
+
+school_cohort_four = FactoryBot.create(:seed_school_cohort, :cip, :valid, :starting_in_2022)
+induction_programme_four = FactoryBot.create(:seed_induction_programme, :cip, school_cohort: school_cohort_four)
+
+school_cohort_five = FactoryBot.create(:seed_school_cohort, :cip, :valid, :starting_in_2022)
+induction_programme_five = FactoryBot.create(:seed_induction_programme, :cip, school_cohort: school_cohort_five)
+
+school_cohort_six = FactoryBot.create(:seed_school_cohort, :fip, :valid, :starting_in_2022)
+induction_programme_six = FactoryBot.create(:seed_induction_programme, :fip, school_cohort: school_cohort_six)
+
+ambition = CoreInductionProgramme.find_by!(name: "Ambition Institute")
+edt = CoreInductionProgramme.find_by!(name: "Education Development Trust")
+ucl = CoreInductionProgramme.find_by!(name: "UCL Institute of Education")
+
+# first some generic mentors
+
+[
+  OpenStruct.new(
+    full_name: "Sally Mentor",
+    email: "rp-mentor.ambition.2021@example.com",
+    core_induction_programme: ambition,
+    school_cohort: school_cohort_one,
+    schedule: Finance::Schedule::ECF.default,
```

Comment:
I think these need to use `Finance::Schedule::ECF.default_for(cohort: xxx)` other wise it will use `Cohort.current`?

Reply:
Good call, will do

Reply:
Is this what you mean @tonyheadford? c58af5f0074ea1